### ISSUE TITLE
Failed to validate png renderer 'null'

### ISSUE THREAD

**Issue by jflambert** [schemaspy/schemaspy#536]
Seeing a renderer warning that I don't see with latest official release 6.0.0. Nothing special about my command-line. Was I supposed to be providing a renderer? Is this a new 6.1.0 requirement, if so the warning should provide the solution. Graphviz 2.40.1-2 is installed

## Expected Behavior
Shouldn't see a warning, or warning should be more helpful.

## Current Behavior
Seeing a renderer warning with the latest schemaspy 6.1.0-SNAPSHOT

## Steps to Reproduce (for bugs)
1. java -jar ./schemaspy-6.1.0-SNAPSHOT.jar -o test -t pgsql -dp ./postgresql-42.2.5.jar -host localhost <your DB params>
2. Observe the following warning
INFO  - Writing/graphing summary
INFO  - Failed to validate png renderer 'null'.  Reverting to default renderer for png.
INFO  - renderer 'null' is not supported by your graphvizVersion of dot
........(25sec)

## Context
* Database type (e.g. MySQL): PostgreSQL 
* Database driver version (is you use a special one): postgresql-42.2.5.jar
* Full command including arguments how you run SchemaSpy: See steps

## Your Environment
* Version used: schemaspy-6.1.0-SNAPSHOT.jar
* Operating System and version: Ubuntu 18.04
* Java version: java -version
openjdk version "10.0.2" 2018-07-17
OpenJDK Runtime Environment (build 10.0.2+13-Ubuntu-1ubuntu0.18.04.4)
OpenJDK 64-Bit Server VM (build 10.0.2+13-Ubuntu-1ubuntu0.18.04.4, mixed mode)

**Comment by npetzall** [comment]
I'll clean up the logging.
From the output above I can't see a warning it's just info.
But it's ugly.

**Comment by jflambert** [comment]
Yes you're absolutely right, it's just green INFO, somehow the words "Failed"/"Reverting"/not supported" resonated to me as warnings. My bad!
